htdocs/js/apps/Essay/essay.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+(() => {
+	const addPreviewButton = (latexEntry) => {
+		if (latexEntry.dataset.previewBtnAdded) return;
+		latexEntry.dataset.previewBtnAdded = 'true';
```

Comment:
Do you want this as the string 'true' or the boolean `true`?  I think it works either way.

Reply:
HTML data attributes can not take boolean values.  It must be a string.

Reply:
Just for clarification, it does work to use a boolean `true` here.  However, the actual type of of the data attribute will be `string`.  All that is really needed in this case is that the data attribute is defined and is "truthy".  You could actually set this to the boolean value of `false`, and the type of the data attribute will still be `string`, and its string value will be 'false'.  Note that is "truthy".  So I deem it better to not use a boolean value as it can be confusing.